framework/doc/content/templates/sqa/app_srs.md.template

```diff
@@ -49,38 +43,50 @@ the [!ac](SRS) for {{app}} is dependent upon the files listed at the beginning o
 !template item key=system-interfaces-requirements
 !sqa requirements link=False collections=SYSTEM category={{category}}
 
-!template item key=human-system-integration
-!include sqa/framework_srs.md start=human-system-integration-begin end=human-system-integration-finish
-
-!template item key=maintainability
-!include sqa/framework_srs.md start=maintainability-begin end=maintainability-finish
+!template! item key=human-system-integration
+{{app}} is a command line driven application which conforms to all standard terminal
+behaviors. Specific human system interaction accommodations shall be a function of the
+end-user's terminal. MOOSE (and therefore {{app}}) does support optional coloring
+within the terminal's ability to display color, which may be disabled.
+!template-end!
 
-!template item key=reliability
-!include sqa/framework_srs.md start=reliability-begin end=reliability-finish
+!template! item key=maintainability
 
-!template item key=information-management
-!include sqa/framework_srs.md start=information-management-begin end=information-management-finish
+- The latest working version (defined as the version that passes all tests in the current regression
+  test suite) shall be publicly available at all times through the repository host provider.
+- Flaws identified in the system shall be reported and tracked in a ticket or issue based system. The
+  technical lead will determine the severity and priority of all reported issues and assign resources
+  at his or her discretion to resolve identified issues.
+- The software maintainers will entertain all proposed changes to the system in a timely manner
+  (within two business days).
+- The core software in its entirety will be made publicly available under the [!ac](LGPL)
+  version 2.1 license.
```

Comment:
This default might need to be adjusted....I guess not all MOOSE-based apps have to be LGPLv2.1?

Reply:
Can we make the license type a template parameter. Or maybe this should be another section.

Reply:
Yeah - a new section makes more sense I think. Some of our controlled apps don't have a standard license name one could just drop in the template.

Reply:
On second thought....I think the section names for this document tie directly into the categories needed. I may just generalize this statement to refer to the LICENSE file unless the "application" is a MOOSE physics module.